backend/api/job.proto

```diff
@@ -243,4 +243,9 @@ message Job {
 
   // Input. Whether the job is enabled or not.
   bool enabled = 16;
+
+  // Optional input field. Whether the job should catch up if behind schedule.
+  // If false, the job will only schedule the latest interval if behind schedule.
+  // If true, the job will catch up on each past interval.
+  bool catchup = 17;
```

Comment:
Is there a way to specify a field in proto as default to true?
Because true is existing behavior, if this becomes default to false, then it will be a breaking change.

Reply:
I can't think of any way other than call it skip_backfill and reverse the logic? 

Also do we need to expose this from UI? via something like a checkbox. If so I think you could choose the default value of such widgets.

Reply:
Yes, we can handle default logic in UI, but I kind of worry someone is calling that from python clients / http endpoints